commercetools-models/src/main/java/io/sphere/sdk/orders/ParcelDraft.java

```diff
@@ -1,23 +1,32 @@
 package io.sphere.sdk.orders;
 
+import io.sphere.sdk.annotations.FactoryMethod;
+import io.sphere.sdk.annotations.ResourceDraftValue;
+
 import javax.annotation.Nullable;
+import java.util.List;
+
 
+@ResourceDraftValue(factoryMethods = {@FactoryMethod(parameterNames = {})})
```

Comment:
Why do we need this annotation when our `ParcelDraftImpl` isn't generated yet?

Reply:
Yep i forgot to replace it by the generated class 😅 , now it's done!